package.json

```diff
@@ -188,7 +188,7 @@
     "postcss": "^8.4.41",
     "postcss-resolve-nested-selector": "^0.1.6",
     "postcss-safe-parser": "^7.0.0",
-    "postcss-selector-parser": "^6.1.1",
+    "postcss-selector-parser": "^6.1.2",
```

Comment:
https://github.com/postcss/postcss-selector-parser/commit/fb38876194130c7518790c116886186fc1d0e37f

Reply:
Nice patch! Once #7928 is merged, please rebase it 🙏🏼